Artiom/src/main/java/CircleArea.java

```diff
@@ -0,0 +1,12 @@
+public class CircleArea {
+    public static void main(String[] args) {
+        System.out.println(calculateCircleArea());
+    }
+    public static double calculateCircleArea() {
+        int radius = 4;
+        double pi = 3.14159265359;
+        double area = pi * (radius * radius);
+        return area;
+    }
+}
+
```

Comment:
Please remove this class from this PR. It is already in your previous PR and will apear here after merge